app/scripts/views/filelist.js

```diff
@@ -41,43 +33,28 @@ export default React.createClass({
           </tr>
         </thead>
         <tbody>
-        {files ? files.map(function (file) {
-          if (typeof file === 'string') file = { id: file }
-
-          var type = '?'
-          if (file.name) {
-            var lastDot = file.name.lastIndexOf('.')
-            if (lastDot !== -1) type = file.name.substr(lastDot + 1, 4).toUpperCase()
+        {files ? files.map(file => {
+          if (typeof file === 'string') {
+            file = { id: file }
           }
 
-          var gatewayPath = t.props.gateway + '/ipfs/' + file.id
-          var dagPath = '#/objects/' + file.id
-
-          var tooltip = (
-            <Tooltip id={file.id}>{i18n.t('Remove')}</Tooltip>
-          )
+          const dagPath = `#/objects/${file.id}`
+          const gatewayPath = `${this.props.gateway}/ipfs/${file.id}`
+          const unpin = (e) => {
+            e.preventDefault()
+            this.props.ipfs.pin.remove(file.id, {r: true}, (err, res) => {
+              if (err) {
+                debug(err.message)
```

Comment:
Do we want to debug log errors we are not properly handling (displaying to the user), or simply just `console.error` them so they are more transparent and can be seen when they occure? I saw this as well in the PR just merged for logging.